WordPress/Classes/Models/ReaderCrossPostMeta.swift

```diff
@@ -1,8 +1,10 @@
 import Foundation
 import CoreData
 
-@objc public class ReaderCrossPostMeta : NSManagedObject
+@objc public class ReaderCrossPostMeta : NSManagedObject, ManagedObject
```

Comment:
Oh fun. :/

In that case, what do you think about moving the extensions into their own files under the `Models` directory in the main project.  At least that way the protocol support would be uniformly implemented in the main project vs split between the main project and the tests. It would also give us the option of using CoreDataHelpers with `ReaderPost` and `SourcePostAttribution` in other places should the need arise.